backend/src/test/java/com/example/backend/service/StorageServiceTest.java

```diff
@@ -1,40 +1,56 @@
 package com.example.backend.service;
-
 import com.example.backend.model.Storage;
 import com.example.backend.repo.StorageRepo;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
-
+import org.mockito.Mockito;
+import org.springframework.mock.web.MockMultipartFile;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.List;
-
 import static org.junit.jupiter.api.Assertions.*;
 import static org.mockito.Mockito.*;
 
 
 class StorageServiceTest {
 
-    StorageRepo repo = mock(StorageRepo.class);
-    StorageService service = new StorageService(repo);
+    StorageRepo storageRepo = mock(StorageRepo.class);
+    StorageService storageService = new StorageService(storageRepo);
 
     @Test
     void getStorages() {
         //GIVEN
-        when(repo.findAll()).thenReturn(List.of());
+        when(storageRepo.findAll()).thenReturn(List.of());
         //WHEN
-        List<Storage> actual = service.getStorages();
+        List<Storage> actual = storageService.getStorages();
         //THEN
         assertEquals(List.of(), actual);
-        verify(repo).findAll();
+        verify(storageRepo).findAll();
     }
 
     @Test
-    void addStorage() {
-    //GIVEN
-        Storage asserted = new Storage("055", "feathers", 49, 32);
-        when(repo.save(asserted)).thenReturn(asserted);
-    //WHEN
-        Storage actual = repo.save(asserted);
-    //THEN
-        assertEquals(asserted, actual);
-        verify(repo).save(asserted);
+    void testAddStorage() throws IOException, URISyntaxException {
+        //GIVEN
+        String id = "your-id";
+        String description = "your-description";
+        int crtsOrg = 10;
+        int crtsNow = 20;
+        MultipartFile imageFile = new MockMultipartFile(
+                "image",
+                "crateGo_logo.png",
+                "image/png",
+                Files.readAllBytes(Paths.get(getClass().getResource("/images/crateGo_logo.png").toURI()))
+        );
+        Storage expectedStorage = new Storage();
+        when(storageRepo.save(any(Storage.class))).thenReturn(expectedStorage);
+        //WHEN
+        Storage createdStorage = storageService.addStorage(id, description, crtsOrg, crtsNow, imageFile);
+        //THEN
+        verify(storageRepo).save(Mockito.any(Storage.class));
+        Assertions.assertEquals(expectedStorage, createdStorage);
```

Comment:
Ist expectedStorage Leer? und createdStorage hat die Werte die du oben in Zeile 51 ihm gegeben hast?